stdlib/public/Reflection/TypeRefBuilder.cpp

```diff
@@ -339,8 +341,10 @@ void TypeRefBuilder::dumpFieldSection(std::ostream &OS) {
         OS << '-';
       OS << '\n';
       for (auto &field : descriptor) {
-        OS << std::string(field.getFieldName(NameOffset).begin(),
-                          field.getFieldName(NameOffset).end());
+        auto Low = sections.ReflectionString.SectionOffset;
+        auto High = sections.ReflectionString.Metadata.size();
+        OS << std::string(field.getFieldName(NameOffset, Low, High).begin(),
+                          field.getFieldName(NameOffset, Low, High).end());
```

Comment:
Given the naming of `High`, I'd expect that to be the address of the end of the section, not the size, so this should be `auto High = Low + sections.ReflectionString.Metadata.size();`

Reply:
Oops, my bad.